x-pack/apm-server/aggregation/servicemetrics/aggregator.go

```diff
@@ -381,6 +442,17 @@ func makeMetricset(key aggregationKey, metrics serviceMetrics) model.APMEvent {
 				Count: metricCount,
 				Sum:   float64(time.Duration(math.Round(metrics.transactionDuration)).Microseconds()),
 			},
+			DurationHistogram: model.Histogram{
+				Counts: counts,
+				Values: values,
+			},
```

Comment:
Now that we have histograms, do we need to keep the `SummaryMetric`?

Reply:
Good question! I think so but my assumption is based on the output document @axw posted in the kibana issue: https://github.com/elastic/kibana/issues/146734#issuecomment-1342467414

Reply:
Yes we still need them, in fact we'll be adding them to transaction metrics too: https://github.com/elastic/apm-server/issues/9655

The reason for having both is that calculating sum, count, and average over histograms is considerably more expensive due to the way they're stored. Elasticsearch ends up having to run through all the buckets of the histogram, vs. looking up one or two sub-fields of an `aggregate_metric_double`.

In the future we should be able to store a histogram along with sum, count, and optionally min & max in a single field: https://github.com/elastic/elasticsearch/issues/81031

Reply:
Ah, that makes sense. Thanks for the clarification, Andrew.